package.json

```diff
@@ -61,7 +61,7 @@
     "test:mocha:report": "lerna run test:mocha --stream --no-bail --no-sort -- -- --timeout 4s --reporter xunit --reporter-option output=nyc/mocha-junit-report.xml",
     "test:realsvc": "lerna run test:realsvc --stream --no-bail --no-sort",
     "test:realsvc:report": "lerna run test:realsvc:report --stream --no-bail --no-sort",
-    "test:report": "npm run test:mocha:report && npm run test:jest:report #&& npm run test:realsvc:report",
+    "test:report": "npm run test:mocha:report && npm run test:jest:report && (true || npm run test:realsvc:report)",
```

Comment:
not sure this work on windows either

Reply:
Good grief...  Sorry about that!  Can't go wrong w/echo...